upgrading/rosa-hcp-upgrading.adoc

```diff
@@ -0,0 +1,29 @@
+:_content-type: ASSEMBLY
+[id="rosa-hcp-upgrading"]
+= Upgrading ROSA with hosted control planes clusters
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-hcp-upgrading
+
+toc::[]
+
+[id="rosa-hcp-lifecycle-policy_{context}"]
+== Life cycle policies and planning
+
+To plan an upgrade, review the xref:../rosa_architecture/rosa_policy_service_definition/rosa-life-cycle.adoc#rosa-life-cycle[{product-title} update life cycle]. The life cycle page includes release definitions, support and upgrade requirements, installation policy information and life cycle dates.
+
+Upgrades are manually initiated or automatically scheduled. Red Hat Site Reliability Engineers (SREs) monitor upgrade progress and remedy any issues encountered.
+
+[id="rosa-hcp-upgrading-a-cluster"]
+== Upgrading a ROSA cluster
+There are three methods to upgrade {{hcp-title-first} clusters:
```

Comment:
should we also specify that control plane and machine pools can be upgraded separately? (and for now only control plane upgrade is supported)

Reply:
I updated this to just the individual upgrading. Can we not upgrade machine pools currently? Your parentheses make me think that it will be supported, but it isn't currently.

Reply:
correct. Currently supported we can only upgrade control plane. Machine pools will come in a next phase

Reply:
Machine pools _can_ be upgraded (sort of) by manually running a `rosa edit machinepool --version=<new-version>`. Unclear if we want to document this now or not

Reply:
Let's wait on that. I see the value, but I prefer not to document an edge case with all these other items in-flight. Can we get a ticket for that update?